packages/cli/src/cache.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ *
+ */
+import fs from 'fs';
+import path from 'path';
+import crypto from 'crypto';
+// $FlowFixMe
+import { mkdirp } from 'mkdirp';
+
+// Default cache directory in `node_modules/.stylex-cache`
+export function getDefaultCachePath() {
+  return process.env.NODE_ENV === 'test'
+    ? path.join('node_modules', '.stylex-cache-test')
+    : path.join('node_modules', '.stylex-cache');
+}
+
+function getCacheFilePath(cachePath, filePath) {
+  const fileName = filePath.replace(/[\\/]/g, '__');
```

Comment:
Can we change the filePath to be relative to the project root instead? (This can be a follow-up PR)

This would make the caches portable and re-usable on different machines.